spec/calendar.html

```diff
@@ -69,7 +69,9 @@ <h1>CalendarMergeFields ( _calendar_, _fields_, _additionalFields_ )</h1>
         1. Let _mergeFields_ be ? GetMethod(_calendar_, *"mergeFields"*).
         1. If _mergeFields_ is *undefined*, then
           1. Return ? DefaultMergeFields(_fields_, _additionalFields_).
-        1. Return ? Call(_mergeFields_, _calendar_, « _fields_, _additionalFields_ »).
+        1. Let _result_ be ? Call(_mergeFields_, _calendar_, « _fields_, _additionalFields_ »).
+        1. Set _result_ to ? ToObject(_result_).
+        1. Return _result_.
```

Comment:
```suggestion
        1. Return ? ToObject(? Call(_mergeFields_, _calendar_, « _fields_, _additionalFields_ »)).
```

Reply:
I do not see otherwhere in the spec which have one line chaining to call a second AO after getting the result from the first AO. IS this allowed in the spec text?

Reply:
It's allowed (you can find various examples in ECMA-262), but not super common.

Reply:
We do it all the time in 262.

The readability of assigning a variable to me is quite poor. It could be split into the call, and a return to object, if having an intermediate variable helps.

Reply:
ok, now I saw one here https://tc39.es/ecma262/#sec-parsefloat-string cool